app/controllers/HasCorrectGoodsController.scala

```diff
@@ -44,23 +45,29 @@ class HasCorrectGoodsController @Inject() (
     extends FrontendBaseController
     with I18nSupport {
 
-  private val form = formProvider()
+  private val form             = formProvider()
+  private val defaultCommodity = Commodity("Default Code", "Default Description")
```

Comment:
is there a chance we display these to the user?

Reply:
No user will not see.

Why I have it first place is I need pass the commodity object to fe

`val commodity = request.userAnswers.get(CommodityCodePage).getOrElse(defaultCommodity)

    Ok(view(preparedForm, mode, commodity))`
I can make it `Commodity("Default Code", "Default Description")`

Instead of `.getOrElse(defaultCommodity)` is there a better way to do this? 

Actually `.getOrElse(defaultCommodity)` is a invalid user scenario.

Reply:
Can we make the view accept an optional commodity Instead and then not do the getOrElse, call me if you want to talk about it.

Reply:
@Chris we made it optional :)